config/initializers/music_paths.rb.sample

```diff
@@ -1,4 +1,4 @@
 # Be sure to restart your server when you modify this file.
 
 # Path to the directory containing your music (TODO: ability to add multiple paths)
-Rails.application.config.music_paths = '/Users/ajk/Music/'
+Rails.application.config.music_paths = '/var/music'
```

Comment:
I think that this file (`config/initializers/music_paths.rb.sample`) could be added on the `.gitignore` file.

Reply:
I think that's just like a `.env.example` file, so we need it there. You mean `music_paths.rb` (without `.sample`)?
```
$ cp config/initializers/music_paths.rb.sample config/initializers/music_paths.rb
```

Reply:
Yeap, this is only the sample file, you need to copy that to run, since the current `music_paths.rb` that will be used on the project can be changed (to set another path)